pkg/Microsoft.ML.FastTree/Microsoft.ML.FastTree.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for FastTree</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
```

Comment:
I assume other nupkgs will now need to be dependent on this new nuget package?  specifically LightGBM?

Reply:
LightGBM probably since it uses RegressionTree from FastTree but are there any other that might be needing this package?

Reply:
Based on our offline conversation, I searched through *.csproj files to see which ones were dependent on fasttree project and for the ones that were and had their own nuget package I added fasttree nuget package as a dependency.